entrypoint.sh

```diff
@@ -23,13 +23,12 @@ fi
 set -euo pipefail
 
 set_tag() {
-if [  "${INPUT_CREATED_TAG}" == "false" ];
-then
-  TAG="$(echo ${GITHUB_REF} | grep tags | grep -o "[^/]*$" || true)";
-else
-  TAG=${INPUT_CREATED_TAG};
-  INPUT_UPDATE_EXISTING="true"
-fi
+  if [ -n "${INPUT_CREATED_TAG}" ]; then
+    TAG=${INPUT_CREATED_TAG}
+    INPUT_UPDATE_EXISTING="true"
```

Comment:
@serucee Why should the release always be updated when a tag is pushed down from the action before?
I would like to remove this.